src/server/getters/get-markets-awaiting-reporting.ts

```diff
@@ -0,0 +1,26 @@
+import * as Knex from "knex";
+import { Address, MarketsRow, UIMarketInfo, UIMarketsInfo, ErrorCallback } from "../../types";
+import { reshapeMarketsRowToUIMarketInfo, getMarketsWithReportingState } from "./get-market-info";
+import { sortDirection } from "../../utils/sort-direction";
+
+// Look up all markets that are currently available for limited reporting.
+// Must be able to sort by number of reports submitted for each market so far, and the response should include the number of reports already submitted -- as well as the payoutNumerator values of each of the reports + the amount staked on each -- as part of the response.
+export function getMarketsAwaitingReporting(db: Knex, reportingWindow: Address|null, reportingRound: number|null, sortBy: string|null|undefined, isSortDescending: boolean|null|undefined, limit: number|null|undefined, offset: number|null|undefined, callback: (err: Error|null, result?: any) => void): void {
```

Comment:
All of the getMarkets* functions (except getMarketsInfo) should return a simple array of market IDs, rather than the big complicated `UIMarketsInfo` object.

(Unless @stephensprinkle has requested otherwise, in which case please do as he asks!)

Reply:
sure thing, easy to change. I'll wait to hear from @stephensprinkle

Reply:
Note that if he hasn't already said otherwise regarding this particular getter, he has expressed that the UI prefers that *all* getters, as a general rule, simply return arrays of market IDs.  So I think that's the desired behavior here as well.

Reply:
👍

Reply:
how do we feel about comma-dangle in javascript? I copied from another augur project, but it didn't have comma-dangle as `always-multiline`, do we want to mimic the same behavior as tslint?

Reply:
Yes, I think `always-multiline` is a good idea!